### PR TITLE
FDN-1807: Update Dockerfile to include Dependence day Dockerfile Contents

### PR DESCRIPTION
**Pull request by himanshuupadhyay101**

### CODE REVIEW

File: Dockerfile-play-builder-17-jammy [hunk 1]
```diff
@@ -1,44 +1,77 @@
 FROM eclipse-temurin:17-jdk-jammy AS builder
 
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Versions
 ARG GIT_USERNAME
 ARG GIT_PASSWORD
-ARG SBT_VERSION
+ARG SBT_VERSION=1.6.2
```
Comment (gheine) [inline Dockerfile-play-builder-17-jammy:8]: sbt version must be passed in from external

File: Dockerfile-play-builder-17-jammy [hunk 2]
```diff
@@ -1,44 +1,77 @@
...
+ARG SCALA_VERSION=2.13
+ARG AMMONITE_VERSION=2.5.2
+ARG APIBUILDER_CLI_VERSION=0.1.42
```
Comment (gheine) [inline Dockerfile-play-builder-17-jammy:11]: please check that this is the latest version

File: Dockerfile-play-builder-17-jammy [hunk 3]
```diff
@@ -1,44 +1,77 @@
...
 
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN apt-get update && \
-    apt-get install --no-install-recommends -qy bash curl libaom3 libavif13 libc6 libcurl4 libfontconfig1 libfreetype6 libgif7 libjpeg8 liblcms2-2 libpng16-16 libssl3 libtiff5 libwebp7 libwebpdemux2 libxml2 postgresql-client ttf-mscorefonts-installer wkhtmltopdf zlib1g && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y --no-install-recommends -qy \
+    bash curl libaom3 libavif13 libc6 libcurl4 libfontconfig1 libfreetype6 libgif7 libjpeg8 liblcms2-2 libpng16-16 libssl3 libtiff5 libwebp7 libwebpdemux2 libxml2 postgresql-client ttf-mscorefonts-installer wkhtmltopdf zlib1g && \
+    ruby golang git awscli hub openssh-server && apt-get clean && rm -rf /var/lib/apt/lists/*
```
Comment (gheine) [inline Dockerfile-play-builder-17-jammy:74]: hard to see what has actually changed here. Can you revert the lines with unchanged packages please so the change clearly shows what was added?

File: Dockerfile-play-builder-17-jammy [hunk 4]
```diff
@@ -1,44 +1,77 @@
...
+ARG AWS_SECRET_ACCESS_KEY
+ARG AWS_SESSION_TOKEN
+
 WORKDIR /root
 
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN apt-get update && \
-    apt-get install --no-install-recommends -qy bash curl libaom3 libavif13 libc6 libcurl4 libfontconfig1 libfreetype6 libgif7 libjpeg8 liblcms2-2 libpng16-16 libssl3 libtiff5 libwebp7 libwebpdemux2 libxml2 postgresql-client ttf-mscorefonts-installer wkhtmltopdf zlib1g && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
```
Comment (gheine) [inline Dockerfile-play-builder-17-jammy:41]: keep?

File: Dockerfile-play-builder-17-jammy (hunk 2 above)
Comment (himanshuupadhyay101) [reply Dockerfile-play-builder-17-jammy]: I have updated the AMMONITE_VERSION and APIBUILDER_CLI_VERSION 
i have not updated the SCALA_VERSION as we are using 2.13 in all the services

**Review by gheine — APPROVED** [review]

**Pull request merged**